docs/stylesheets/language-tags.css

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Supported language tag styling  */
+
+.language-support-tag {
+ display: inline-block;
+ font-family: Roboto-Regular, Roboto, Arial, sans-serif;
+ font-size: 0.6em;
+ font-weight: 400;
+ line-height: 1em;
+ color: white;
+ border-radius: 0.2em;
+ overflow: hidden; /* Ensures the border-radius is applied to the child elements */
+}
+.supported {
+ background-color: #555555;
+ padding: 4px 6px;
+ display: inline-block;
+}
+.python {
+ background-color: #007ec6;
+ padding: 4px 6px;
+ display: inline-block;
+}
+.java {
+ background-color: #ED8B00;
+ color: black;
+ padding: 4px 6px;
+ display: inline-block;
+}
+.go {
+ background-color: #00ADD8;
+ color: black;
+ padding: 4px 6px;
+ display: inline-block;
+}
+.typescript {
+ background-color: #235A97;
+ padding: 4px 6px;
+ display: inline-block;
+}
+.preview {
```

Comment:
We might want to change this class name to `experimental` so we don't conflict with other definitions of preview that might clash with preview features in Cloud. For example,

> The Agent Engine Code Execution feature is a Preview release. For more information, see the [launch stage descriptions](https://cloud.google.com/products#product-launch-stages).